packages/terraform-modules/README.md

```diff
@@ -0,0 +1,20 @@
+# Terraform Testnets
+
+## Overview
+
+Terraform is a tool that allows developers to treat "infrastructure as code."
+Infrastructure is defined in modules, and Terraform creates/changes/destroys
+when changes are applied.
+
+## Local Setup
+
+It's best to use this package with `celotool`, but if you need to
```

Comment:
I assume we need people to download terraform to run these commands? Maybe we should add that documentation somewhere. Or ideally install it on demand when running the commands for the first time

Reply:
Here's a PR for the dev setup with a Terraform note: https://github.com/celo-org/bootnode/pull/62

And an issue for auto-downloading in the future: https://github.com/celo-org/celo-monorepo/issues/442